src/utils/index.py

```diff
@@ -210,6 +220,12 @@ def main() -> None:
         help="Path to the samples to be indexed, as seen by ursadb (if different).",
         default=None,
     )
+    parser.add_argument(
+        "--min-file-size-mb",
+        type=int,
+        help="Minimum file size, in MB, to index. 0 By default.",
+        default=0,
```

Comment:
Maybe 'indexing empty files' is not the best default? 🤔

Reply:
it's not wrong though (principle of least surprise suggests that we shouldn't ignore any file in the directory)